src/aioquic/h3/connection.py

```diff
@@ -128,11 +129,11 @@ def encode_settings(settings: Dict[int, int]) -> bytes:
     return buf.data
 
 
-def parse_max_push_id(data: bytes) -> int:
+def parse_id(data: bytes) -> int:
```

Comment:
Maybe `parse_uint_var` would be a better name?

Reply:
agree

Reply:
This function is used when you have some bytes which contain *only* a variable-length integer, hence the EOF check, to check all bytes have been consumed.